src/pages/study/views/Publication.js

```diff
@@ -38,7 +38,7 @@ const Publication = ({
                         :
                       </Grid>
                       <Grid item xs={12} sm={6} className={classes.content}>
-                        <a href={getURL(publication[attr.key], attr.url)} target="_blank" rel="noreferrer" className={classes.outLink}>
+                        <a href={getURL(publication[attr.key], attr.url)} target="icdc" className={classes.outLink}>
```

Comment:
@rana22 looks like this change is reverting #331. Can you have look into ?

Reply:
I reverted this change.